modules/authorization/src/main/java/com/intuit/wasabi/authorization/Authorization.java

```diff
@@ -127,4 +127,28 @@
      * @return the complete {@link UserInfo}
      */
     UserInfo getUserInfo(UserInfo.Username userID);
```

Comment:
@abhishekjain88 Since this interface is updated with superadmin methods, you will need to update the code in wasabi-intuit's IUSAuthorizationImpl as well. It won't compiled once wasabi-intuit is pointing to this.